python/mujinplanningclient/realtimerobotplanningclient.py

```diff
@@ -1020,19 +1443,101 @@ def SetRobotBridgePause(self, timeout=10, **kwargs):
 
     def SetRobotBridgeResume(self, timeout=10, **kwargs):
         """
-
         Args:
             timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
+            unit (str, optional): The unit of the given values. (Default: 'mm')
+            robotname (str, optional): Name of the robot
+            toolname (str, optional): Name of the manipulator. Defaults to currently selected tool
+            robotBridgeConnectionInfo (dict, optional): Information to set up a client to the robot bridge.
+            locationCollisionInfos (dict, optional): List of external collision IOs to be computed and sent in realtime.
         """
         taskparameters = {
             'command': 'SetRobotBridgeResume',
         }
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def SetRobotBridgeServoOn(self, servoon, robotname=None, timeout=3, fireandforget=False):
+    def SetJogModeVelocities(self, movejointsigns, robotname=None, toolname=None, robotspeed=None, robotaccelmult=None, canJogInCheckMode=None, timeout=1, fireandforget=False, jogtype=_deprecated, checkSelfCollisionWhileJogging=None, force=None, unit='mm', robotBridgeConnectionInfo=None, locationCollisionInfos=None, robotJogParameters=None, simulationtimestep=None, plotDirection=None, **kwargs):
```

Comment:
What happened here?

Reply:
I accidentally messed up the method order. Not sure how it slipped into the MR. Sorry. Fixed.